packages/react-icons/convert.js

```diff
@@ -1,10 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-const svgr = require("@svgr/core");
```

Comment:
Remove `svgr` because now we only want to extract data but never transform the SVG.